### PR TITLE
feat(oras): Remove helm cli dependency

### PR DESCRIPTION
**Pull request by tompizmor**
This PR deletes the `helmcli` package and so the helm cli requirement. Previously we were only using it for pushing charts to  OCI repositories, from now on, we can use oras library directly.

Fixes https://github.com/bitnami-labs/charts-syncer/issues/73

### CODE REVIEW

**Review by Javirln — APPROVED** [review]
LGTM!

File: pkg/syncer/sync.go [hunk 1]
```diff
@@ -106,6 +92,17 @@ func (s *Syncer) SyncPendingCharts(names ...string) error {
...
+		// Read final chart metadata
+		configFilePath := fmt.Sprintf("%s/Chart.yaml", chartPath)
+		chartConfig, err := ioutil.ReadFile(configFilePath)
+		if err != nil {
+			return err
+		}
+		metadata := &helmchart.Metadata{}
+		if err = yaml.Unmarshal(chartConfig, metadata); err != nil {
+			return err
+		}
```
Comment (jotadrilo) [inline pkg/syncer/sync.go:104]: note that we are in a loop, let's use `multierror` and continue

```suggestion
		// Read final chart metadata
		configFilePath := fmt.Sprintf("%s/Chart.yaml", chartPath)
		chartConfig, err := ioutil.ReadFile(configFilePath)
		if err != nil {
			klog.Errorf("unable to read %q metadata: %+v", id, err)
			errs = multierror.Append(errs, errors.Trace(err))
			continue
		}
		metadata := &helmchart.Metadata{}
		if err = yaml.Unmarshal(chartConfig, metadata); err != nil {
			klog.Errorf("unable to decode %q metadata: %+v", id, err)
			errs = multierror.Append(errs, errors.Trace(err))
			continue
		}
```

File: pkg/client/oci/oci.go [hunk 2]
```diff
@@ -289,14 +288,32 @@ func (r *Repo) Upload(file, name, version string) error {
 		return errors.Trace(err)
 	}
 
-	chartRef := fmt.Sprintf("%s%s/%s:%s", r.url.Host, r.url.Path, name, version)
-	if err := helmcli.SaveOciChart(file, chartRef); err != nil {
-		return errors.Trace(err)
+	memoryStore := content.NewMemoryStore()
+	var resolver = r.newDockerResolver()
```
Comment (jotadrilo) [inline pkg/client/oci/oci.go:292]:
```suggestion
	resolver := r.newDockerResolver()
```

**Review by jotadrilo — APPROVED** [review]
I left minor comments, please tackle them before your merge the PR!
Good work here, LGTM!

**Pull request merged**